Code/Framework/AzCore/AzCore/Memory/SystemAllocator.h

```diff
@@ -51,7 +50,7 @@ namespace AZ
                 , m_allocationRecords(true)
                 , m_stackRecordLevels(5)
             {}
-            IAllocatorAllocate*         m_custom;   ///< You can provide our own allocation scheme. If NULL a HeapScheme will be used with the provided Descriptor.
+            IAllocatorSchema*         m_custom;   ///< You can provide our own allocation scheme. If NULL a HeapScheme will be used with the provided Descriptor.
```

Comment:
repeat: Is the `m_custom` member needed, if the base `IAllocator` class already has a `m_schema` IAllocatorSchema* member already?